focus-areas/health-101/README.md

```diff
@@ -0,0 +1,19 @@
+# Sustainability
+
+Goal: We have a set of light-weight metrics and methods required to evaluate the very basic health of an open source project.
+
+# Some Potential User Stories
+
+
+* As a potential consumer of an open source project I want to evaluate the health of engagement around that project before ingesting/using.
+* As someone contributing to open source, I want to assess the health of the project before I invest amy of my time there.
+* As a maintainer of a project, I want to make sure that I have covered all of the basics to ensure my project is setup for success, and track that progress over time
```

Comment:
Maybe add one more role: Open source foundations to support projects and communities by evaluating its health, to decide in which level support needed, like financial resources etc.